src/us/kbase/assemblyhomology/core/Namespace.java

```diff
@@ -101,11 +112,12 @@ public String getSourceDatabaseID() {
 	public int hashCode() {
```

Comment:
It would be useful to have a docstring for this function

Reply:
That's the standard hashCode() fn overridden from Object, so it inherits docs from there.

Reply:
Usually hashCode isn't shown in the built docs though.